conftest.py

```diff
@@ -7,9 +7,11 @@
 seconds.
 """
 
+
 def pytest_addoption(parser):
     parser.addoption("--expensive", action="store_true",
-            help="run expensive tests (which are otherwise skipped).")
```

Comment:
I'm curious -- did you use PyCharm's autoformat tool?

Reply:
Yep